src/language-server/document.ts

```diff
@@ -66,6 +66,8 @@ export function extractGraphQLDocuments(
     case "typescriptreact":
     case "vue":
       return extractGraphQLDocumentsFromJSTemplateLiterals(document, tagName);
+    case "svelte":
+      return extractGraphQLDocumentsFromJSTemplateLiterals(document, tagName);
```

Comment:
Lets fold this into the existing `extractGraphQLDocumentsFromJSTemplateLiterals` cases
```suggestion
    case "vue":
    case "svelte":
        return extractGraphQLDocumentsFromJSTemplateLiterals(document, tagName);
```